### ISSUE TITLE
./examples/http_no_tls.sh has errors.

### ISSUE THREAD

**Issue by sdavis-r7** [sdavis-r7/http2-probe#8]
<img width="484" alt="screen shot 2016-02-04 at 4 41 30 pm" src="https://cloud.githubusercontent.com/assets/16342638/12834678/2acdbd8e-cb5e-11e5-8662-c765e3d22ab8.png">

**Comment by sdavis-r7** [comment]
HTTP (80/TCP) throws error with HTTP/2 handshake.  see screenshot.

**Comment by jhart-r7** [comment]
While hidden, the error is actually coming from the remote server.  If you fire up wireshark and look at the traffic for one of these IPs, you'll see the `FRAME_SIZE_ERROR`.  Googling around I found  https://http2.github.io/http2-spec/#FrameSize.  In examining packet captures I obtained when testing two different hosts, both hosts sent back somewhat unexpected HTTP messages (frames?) immediately after the `SETTINGS,HEADERS`.  One sent back a 301, the other 50x.  After that is where the `FRAME_SIZE_ERROR` occurs.
